site/app/themes/[slug]/page.tsx

```diff
@@ -87,7 +87,8 @@ export default async function Page(props: ThemePageProps) {
 
           <h3 className="text-2xl font-semibold mb-1">Use theme</h3>
 
-          <h4 className="text-lg font-medium mb-1">Pick your media type or platform</h4>
+          <h4 className="text-lg font-medium mb-0">Pick your media type or player</h4>
+          <p className="mb-1" style={{color: "#777"}}>Are we missing something?<a className="underline" href="https://github.com/muxinc/player.style/issues" target='_blank'>Tell us.</a></p>
```

Comment:
there is no space in between `something?` and `Tell us`.

instead of a plain gray 777 could you try using the class `gray` or `gray-100` to `gray-700`?
that makes it more complement the site colors, using the Tailwind guardrails basically.

Reply:
It doesn't look like `gray`s are implemented. Or at least they're not doing anything for me. Gonna merge this but happy to update when those are supported.